### PR TITLE
Adding enumerator instead of static final integers

### PR DESCRIPTION
**Pull request by somalieca**
As stated in the book Effective Java on page 147, the use of enumerators is way better than static.

### CODE REVIEW

**Comment by chesterbr** [comment]
Hi! Thank you for your contribution @somalieca. Indeed, Enumerators are a better choice for this case (they didn't exist in Java when this code was created). However, there are other references to those constants in the project. Before I can actually merge, could you please update them as well (and ensure the code properly builds?). Thank you!

**Comment by chesterbr** [comment]
Another thing: the original list of ordered "napies" (card suits, in Portuguese) did not include "nenhum" (none, in Portuguese). Is this inclusion affecting the gameplay?

**Comment by somalieca** [comment]
Hi, the inclusion of none(nenhum) in the enum doesnt affect the gameplay, the logic is still the same. I updated the other files too.

File: app/build.gradle [hunk 1]
```diff
@@ -2,7 +2,7 @@ apply plugin: 'com.android.application'
 
 android {
     compileSdkVersion 8
-    buildToolsVersion "23.0.2"
+    buildToolsVersion '25.0.0'
```
Comment (chesterbr) [inline app/build.gradle]: Is this update necessary?

File: app/src/main/java/me/chester/minitruco/core/EstrategiaGasparotto.java [hunk 2]
```diff
@@ -127,8 +127,7 @@ private boolean mandaBala(int fatorDeDuvida) {
 			fatorDeDuvida = 100;
 		if (fatorDeDuvida < 0)
 			fatorDeDuvida = 0;
-		return (Math.abs(rand.nextInt()) % 100 + 1 <= fatorDeDuvida) ? true
-				: false;
+		return (rand.nextInt(100) + 1) <= fatorDeDuvida;
```
Comment (chesterbr) [inline app/src/main/java/me/chester/minitruco/core/EstrategiaGasparotto.java]: Thank you for cleaning up those.

File: app/src/main/java/me/chester/minitruco/core/EstrategiaGasparotto.java [hunk 3]
```diff
@@ -621,8 +620,8 @@ else if (carta.getValorTruco(s.manilha) == 14)
 		// utilizada diretamente
 		// Estas são cartas fictícias somente para os testes condicionais logo
 		// abaixo
-		Carta tres_testedemanilha = new Carta('3', 3);
-		Carta dois_testedemanilha = new Carta('2', 3);
+		Carta tres_testedemanilha = new Carta('3', Carta.Naipe.get(3));
+		Carta dois_testedemanilha = new Carta('2', Carta.Naipe.get(3));
```
Comment (chesterbr) [inline app/src/main/java/me/chester/minitruco/core/EstrategiaGasparotto.java]: Maybe we could just say `Carta.Naipe.PAUS`?

(the original code should have used the constant anyway)

File: app/src/main/java/me/chester/minitruco/core/JogoLocal.java [hunk 4]
```diff
@@ -272,9 +272,9 @@ private void iniciaMao(Jogador jogadorQueAbre) {
 	 * Processa uma jogada e passa a vez para o próximo jogador (ou finaliza a
 	 * rodoada/mão/jogo), notificando os jogadores apropriadamente
 	 * 
-	 * @param j
+	 *
```
Comment (chesterbr) [inline app/src/main/java/me/chester/minitruco/core/JogoLocal.java]: good catch!

File: build.gradle [hunk 5]
```diff
@@ -4,7 +4,7 @@ buildscript {
         jcenter()
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:1.3.0'
+        classpath 'com.android.tools.build:gradle:2.3.3'
```
Comment (chesterbr) [inline build.gradle]: [Same above] Is this update necessary?

File: gradle/wrapper/gradle-wrapper.properties [hunk 6]
```diff
@@ -1,6 +1,6 @@
-#Wed Apr 10 15:27:10 PDT 2013
+#Mon Jul 03 23:01:24 EEST 2017
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-2.2.1-all.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-3.3-all.zip
```
Comment (chesterbr) [inline gradle/wrapper/gradle-wrapper.properties]: [Same above] Is this update necessary?

**Comment by chesterbr** [comment]
This became out of sync with the big changes introduced lately, so I'll close for now. Thank you for PR-ing 🙇

**Pull request closed without merging**